### PR TITLE
feat: add standalone demo

### LINKED ISSUE #671: Re-write the demo docs for both cluster and standalone
- detailed the demo instructions
- cluster and standalone use the same scene/story

### LINKED ISSUE #733: Add standalone demo
* make a predict-taxi-trip-duration demo with standalone mode

### PR DESCRIPTION
**Pull request by dl239**

### CODE REVIEW

File: demo/predict-taxi-trip-duration-nb/demo/README.md [hunk 1]
```diff
@@ -48,5 +48,88 @@ python3 predict.py
...
+python3 predict.py
+# the output we will see
+----------------ins---------------
+[[ 2.       40.774097 40.774097 40.774097 40.774097 40.774097 40.774097
+  40.774097 40.774097  1.        1.      ]]
+---------------predict trip_duration -------------
+880.3688347542294 s
+```
+
+To read more details about standalone mode, please refer [here](https://github.com/4paradigm/OpenMLDB/blob/main/docs/en/standalone.md)
```
Comment (zhanghaohit) [inline demo/predict-taxi-trip-duration-nb/demo/README.md]: Could you also add the refer link for cluster doc?

File: demo/predict-taxi-trip-duration-nb/demo/openmldb_batch.py [hunk 2]
```diff
@@ -1,21 +0,0 @@
-#! /usr/bin/env python3
```
Comment (zhanghaohit) [inline demo/predict-taxi-trip-duration-nb/demo/openmldb_batch.py:1]: as this file is removed. We have to update the [quickstart](docs/en/cluster.md)

**Review by zhanghaohit — CHANGES_REQUESTED** [review]

File: demo/predict-taxi-trip-duration-nb/demo/README.md [hunk 3]
```diff
@@ -48,5 +48,89 @@ python3 predict.py
...
+## Running the demo with standalone mode
+Start docker
+```bash
+docker run -it ghcr.io/4paradigm/openmldb:0.3.0 bash
+```
+Initilize environment
+```bash
+./init.sh standalone
 ```
+Create table and import the data to OpenMLDB. The recent data will be imported again in the real scene before deploying online service.
```
Comment (lumianph) [inline demo/predict-taxi-trip-duration-nb/demo/README.md]: remove "The recent data will be imported again in the real scene before deploying online service." here. We will explain this when deploying the online sql

File: demo/predict-taxi-trip-duration-nb/demo/README.md [hunk 4]
```diff
@@ -48,5 +48,89 @@ python3 predict.py
...
+min(pickup_latitude) over w2 as pc_min_pl,
+avg(pickup_latitude) over w2 as pc_avg_pl ,
+count(vendor_id) over w2 as pc_cnt,
+count(vendor_id) over w as vendor_cnt
+from t1
+window w as (partition by vendor_id order by pickup_datetime ROWS_RANGE BETWEEN 1d PRECEDING AND CURRENT ROW),
+w2 as (partition by passenger_count order by pickup_datetime ROWS_RANGE BETWEEN 1d PRECEDING AND CURRENT ROW);
+> quit
+```
+
```
Comment (lumianph) [inline demo/predict-taxi-trip-duration-nb/demo/README.md]: you can add the below description here:
Note that for a real-world application, the user may import another copy of recent data for online inference before SQL deployment (refer to the cluster-mode demo). For the sake of simplicity, this demo just uses the same data for both offline training and online inference.

**Review by lumianph — CHANGES_REQUESTED** [review]

**Review by zhanghaohit — APPROVED** [review]

**Review by lumianph — APPROVED** [review]

**Pull request merged**